src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java

```diff
@@ -159,4 +164,35 @@
 
     }
 
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class IslandDamageSettings {
+
+        public boolean pvpOnIslands = true;
+        public boolean pvpBetweenMembers = false;
+        public boolean mobsVisitorTargeting = false;
+
+        public List<String> membersPreventedDamages = new ArrayList<String>() {{
```

Comment:
Arrays.asList

Reply:
You will almost never want anything containing `{{` because it generates a lot of unnecessary stuff